scripts/linux/github/updateLatestVersion.sh

```diff
@@ -0,0 +1,257 @@
+        proposedImageVersion=$(cat $IOTEDGE_REPO_PATH/versionInfo.json | jq ".version" | tr -d '"')
+        proposedIisVersion=$(grep "PACKAGE_VERSION:" $IIS_REPO_PATH/.github/workflows/packages.yaml | awk '{print $2}' | tr -d "'" | tr -d '"')
+
+        contentIe=$(cat $TARGET_IE_FILE)
+        latestEdgeletVersion=$(echo $contentIe | jq '."aziot-edge"' | tr -d '"')
+        contentIis=$(cat $TARGET_IIS_FILE)
+        latestIisVersion=$(echo $contentIis | jq '."aziot-identity-service"' | tr -d '"')
+
+        # Verify
+        echo "Sanity check iotedge version"
+        version_sanity_check $proposedEdgeletVersion $latestEdgeletVersion
+        echo "Sanity check docker image version"
+        version_sanity_check $proposedImageVersion get_latest_release_per_branch_name
+        echo "Sanity check iot-identity-service version"
+        version_sanity_check $proposedIisVersion $latestIisVersion
+
+        # Update the version files
+        jqQuery=".\"aziot-edge\" = \"$proposedEdgeletVersion\""
+        echo $contentIe | jq "$jqQuery" > $TARGET_IE_FILE
+        jqQuery=".\"aziot-identity-service\" = \"$proposedIisVersion\""
+        echo $contentIis | jq "$jqQuery" > $TARGET_IIS_FILE
+        
+        # Pring log for debugging
+        echo "Update $TARGET_IE_FILE:"
+        cat $TARGET_IE_FILE | jq '.'
+        echo ""
+        echo "Update $TARGET_IIS_FILE:"
+        cat $TARGET_IIS_FILE | jq '.'
+
+    elif [ "$BRANCH_NAME" == "refs/heads/main" ]; then
+        echo "I'm pretty sure you don't want to release from the main branch."
+        exit 1;
+    else
+        echo "Oh dear, how did you get here?!?"
+        echo "Let me not let you do the release from your pull request branch"
+        exit 1;
+    fi
+}
+
+
+#######################################
+# NAME: 
+#    github_update_and_push
+# DESCRIPTION:
+#    The function commits the changes, tag, and push them to remote repository
+# GLOBALS:
+#    AZURE_IOTEDGE_REPO_PATH ____ Path to /azure-iotedge        (string)
+#        i.e. $(Build.SourcesDirectory)/azure-iotedge
+#    VERSION ____________________ Current release version       (string)
+#    GITHUB_PAT _________________ Github Personal Access Token  (string)
+# OUTPUTS:
+#    Updated latest version JSON files
+# REMARK:
+#    Please make sure the directories provided need to have a proper branch/commit 
+#    checked out.
+#######################################
+github_update_and_push()
+{
+    [[ -z "$AZURE_IOTEDGE_REPO_PATH" ]] && { echo "\$IOTEDGE_REPO_PATH is undefined"; exit 1; }
+    [[ -z "$VERSION" ]] && { echo "\$VERSION is undefined"; exit 1; }
+    [[ -z "$GITHUB_PAT" ]] && { echo "\$GITHUB_PAT is undefined"; exit 1; }
+
+    cd $AZURE_IOTEDGE_REPO_PATH
+    git config user.name iotedge1
+    git commit -am "Prepare for Release $VERSION"
+    lastCommitHash=$(git log -n 1 --pretty=format:"%H")
+    git tag "$VERSION" $lastCommitHash
+    git push https://$GITHUB_PAT@github.com/Azure/azure-iotedge.git
```

Comment:
We should find a more secure way to do this (where the PAT isn't part of the URL). Soon.